Projects/App/Sources/Screens/BankIDLoginScreen/SimpleSignLogin.swift

```diff
@@ -0,0 +1,117 @@
+import Apollo
+import Flow
+import Form
+import Foundation
+import hCore
+import hCoreUI
+import hGraphQL
+import Presentation
+import UIKit
+
+struct SimpleSignLoginView: Presentable {
+    @Inject var client: ApolloClient
+
+    func materialize() -> (UIViewController, FiniteSignal<String>) {
+        let viewController = UIViewController()
+        viewController.preferredPresentationStyle = .modal
+
+        let bag = DisposeBag()
+
+        let form = FormView()
+
+        let masking = Localization.Locale.currentLocale.market.masking
+
+        viewController.title = L10n.bankidLoginTitle
+
+        let label = UILabel()
+        label.text = Localization.Locale.currentLocale.market.labelTitle
+
+        let textField = UITextField()
```

Comment:
What do you think about making a utility func on masking that applies it to a `UITextField`, something like `Masking.applyTo(_ textField: UITextField) -> Bag` which handles `isValid`, `textContentType`, `keyboardType`, etc.

Reply:
How would you add `isValid` in this case? Perhaps it needs to be thought out a bit more before we add a convenience function for it.

Reply:
I guess `maskValue` would make sense, although I am not really using this. Actually it's never really masking the value is it? Just formatting it.

Reply:
I tried doing this a few ways, but I don't really like the idea of a method returning a signal for validity in a masking object. Do you think we should split up `masking` and `validation` -> perhaps we can change `maskType` to something like `textFieldType` or `textFieldContentType` ?

Reply:
Also the textField settings would be configurable from the type itself perhaps? Then masking would just have an `applyMask` and `validation` would have a `isValidSignal`

Reply:
@tarikstafford don't we always wanna mask? it makes sure that the user can only enter the valid characters, also it inserts the hyphens at the correct places and so forth also we have the same predictable behaviour in all places of the app. And then we can just use `unmaskValue` before sending to backend.